### ISSUE TITLE
Error occured when I use "/rpginv open MyName"

### ISSUE THREAD

**Issue by AndyYu168** [EndlessCodeGroup/RPGInventory#107]
When I execute command "/rpginv open myname" and put an armor into my empty slot, an error occured on my console and I can take the slot-item out.
(This only happens on myself, If I open other players' inventory this will not happen.)
Here is the log, please fix it! Thank you!

```AndyYu168 issued server command: /rpginv open AndyYu168
Could not pass event InventoryClickEvent to RPGInventory v2.1.4
org.bukkit.event.EventException: null
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot-1.12-R0.1-SNAPSHOT-b1299.jar:git-Spigot-ede465d-bb4ae3b]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot-1.12-R0.1-SNAPSHOT-b1299.jar:git-Spigot-ede465d-bb4ae3b]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:499) [spigot-1.12-R0.1-SNAPSHOT-b1299.jar:git-Spigot-ede465d-bb4ae3b]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:484) [spigot-1.12-R0.1-SNAPSHOT-b1299.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1877) [spigot-1.12-R0.1-SNAPSHOT-b1299.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.PacketPlayInWindowClick.a(SourceFile:33) [spigot-1.12-R0.1-SNAPSHOT-b1299.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.PacketPlayInWindowClick.a(SourceFile:10) [spigot-1.12-R0.1-SNAPSHOT-b1299.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.PlayerConnectionUtils$1.run(SourceFile:13) [spigot-1.12-R0.1-SNAPSHOT-b1299.jar:git-Spigot-ede465d-bb4ae3b]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_151]
	at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_151]
	at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46) [spigot-1.12-R0.1-SNAPSHOT-b1299.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:747) [spigot-1.12-R0.1-SNAPSHOT-b1299.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:405) [spigot-1.12-R0.1-SNAPSHOT-b1299.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:678) [spigot-1.12-R0.1-SNAPSHOT-b1299.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576) [spigot-1.12-R0.1-SNAPSHOT-b1299.jar:git-Spigot-ede465d-bb4ae3b]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_151]
Caused by: java.lang.NullPointerException
	at org.bukkit.event.inventory.InventoryClickEvent.<init>(InventoryClickEvent.java:63) ~[spigot-1.12-R0.1-SNAPSHOT-b1299.jar:git-Spigot-ede465d-bb4ae3b]
	at ru.endlesscode.rpginventory.event.listener.InventoryListener.onArmorSlotClick(InventoryListener.java:418) ~[?:?]
	at ru.endlesscode.rpginventory.event.listener.InventoryListener.onInventoryClick(InventoryListener.java:321) ~[?:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_151]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_151]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_151]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_151]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:302) ~[spigot-1.12-R0.1-SNAPSHOT-b1299.jar:git-Spigot-ede465d-bb4ae3b]
	... 15 more```

**Comment by Fargus5** [comment]
This was not already be fixed when the 2.1.4 version was relased?

**Comment by KnifeKat** [comment]
One fix at a time. Have patience. 

Instead of badgering @OsipXD , @Fargus5 , you should respect the Developer and give him time to run through the issues and fix them. If he waited to fix all the issues all at once, he would not have a better updated version for you at all.

Might as well release updates to fix a few things at a time, so those issues are taken care of instead of leaving things broken and make you wait till -everything- is fixed, before giving you a better updated version. 

This will be dealt with in due time.

Thank you in advance.

**Comment by Fargus5** [comment]
@KnifeKat where do you see the lack of respect?

**Comment by AndyYu168** [comment]
> Might as well release updates to fix a few things at a time, so those issues are taken care of instead of leaving things broken and make you wait till -everything- is fixed, before giving you a better updated version.

@KnifeKat Can't agree more, maybe the author can make a small update about fixing a tiny bug, we will be happy cuz we know that the author doesnt abandon the plugin, he is busy fixing it and make it better.

As for @Fargus5  This bug was found by me, I'm waiting for the author to fix it, if this bug disturbs you too please be patient, say no more please.

**Comment by Fargus5** [comment]
ok thanks Andy

**Comment by OsipXD** [comment]
Was fixed in #108
